### DISCUSSION TITLE
Does Twitch GUI 2.0 not support Streamlink 3.1.1 (any 3.x)?

### DISCUSSION THREAD

**Discussion by lorthris** [streamlink/streamlink-twitch-gui#873] [Q&A] [answered]
Hi All,

 I have uninstalled GUI 1.13 and streamlink 2.3 (the latest versions of both available chocolately btw - i would much prefer to continue using choco) as the twitch API has changed and I need GUI 2.0

I have installed GUI 2.0 and Streamlink 3.1.1 from github using the .exe (latest versions of both)

I now get the below error when I go to launch a stream.
![image](https://user-images.githubusercontent.com/12410350/155285990-1be77bdb-2c0d-4deb-89db-2adde50e8a6c.png)

From what I can tell, these -rtmp arguemnts were removed/ no longer supported in streamlink 3.0 onwards
https://streamlink.github.io/changelog.html#streamlink-3-0-0-2021-11-17

So does GUI 2.0 not support 3.0 at all? seems it is trying to do these calls? and then it has 2.4 as a minimum now. So it ONLY supports 2.4? I can confirm I am running 2.0 and 3.1.1
![image](https://user-images.githubusercontent.com/12410350/155286418-260d0ab7-fe74-455b-b07d-442e79afb23c.png)

![image](https://user-images.githubusercontent.com/12410350/155286484-a8ca41ef-9d09-48ac-a7ed-a8d5beba429e.png)

Thanks for your good work so far and for any help you provide.
James

**Comment by lorthris** [comment]
update - i have done the same thing on another pc (uninstalled 1.13 and 2.3 and updated to 2.0 and 3.1.1 and it works - must be something that didnt uninstall correctly on the initial pc. (the only difference i can think of is I uninstalled streamlink with revo on this PC and using choco uninstall with the initial pc)

**Comment by bastimeyer** [comment, marked as answer]
https://github.com/streamlink/streamlink-twitch-gui/issues/842#issuecomment-973369948
